CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ Bug-fixes
 Non-backwards compatible changes
 --------------------------------
```

Comment:
I think it's possibly worth adding a CHANGELOG entry mentioning that the dependencies of certain key modules have been significantly reduced? We should at least get some credit for all the hard work we do under the surface :smile: